apps/adoption/src/app/templates/angular/styles.css.txt

```diff
@@ -0,0 +1,3 @@
+body {
+  padding: 1rem;
```

Comment:
What's up with this text file?

Reply:
This is for one reason only - presentation. 

When you use the embed option for Stackblitz or Codesandbox and even when you open them in their own interfaces - standalone components are displayed stuck to top-left corner so sometimes even their own border is hard to see. So this `padding: 1rem` is just giving some space so whatever we try to demo is more clearly visible - in demo mode.